service/history/service.go

```diff
@@ -214,7 +214,8 @@ func (s *Service) Start() {
 		p.CassandraConfig.Keyspace,
 		s.config.ExecutionMgrNumConns,
 		p.Logger,
-		base.GetMetricsClient())
```

Comment:
What does this method do? Should it be removed if no longer used?

Reply:
this is actually referenced at line 138:
https://github.com/uber/cadence/pull/484/files/b41eff2eeeca590eba88c5cb7ef67de7977ac034#diff-48531afaf74492193b35d730980b6ecdR138

Reply:
@madhuravi 
this is actually referenced at line 138:
https://github.com/uber/cadence/pull/484/files/b41eff2eeeca590eba88c5cb7ef67de7977ac034#diff-48531afaf74492193b35d730980b6ecdR138